.buildkite/hooks/pre-command

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+echo "~~~ setting docker image tag"
+case "${BUILDKITE_BRANCH}" in
+  develop)
+    IMAGE_TAG="latest"
+    ;;
+  *)
+    if [ "$BUILDKITE_PULL_REQUEST" != "false" ]; then
+      IMAGE_TAG="PR-$BUILDKITE_PULL_REQUEST"
+    else
+      IMAGE_TAG="testing"
```

Comment:
Shouldn't this be unique as well? Or we don't care if they step on each other unless it's a PR?

Reply:
This could be `stellargraph-public-stellargraph-build-${BUILDKITE_BUILD_NUMBER}` so it's the same auxiliary image that's pushed anyway?

Reply:
This is currently just preserving existing behaviour; but I can change it here too.